Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionMicrosoftSysmonWindowsEvent.yaml

```diff
@@ -0,0 +1,139 @@
+Parser:
```

Comment:
we don't have Im parser. Need to add that too

Reply:
done